homeassistant/components/maxcube/binary_sensor.py

```diff
@@ -1,8 +1,10 @@
 """Support for MAX! binary sensors via MAX! Cube."""
 from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_BATTERY,
     DEVICE_CLASS_WINDOW,
     BinarySensorEntity,
 )
+from homeassistant.const import ENTITY_CATEGORY_DIAGNOSTIC
```

Comment:
Can you use the new EntityCategory enumerator?

Reply:
Done https://github.com/home-assistant/core/pull/58699/commits/cfb0f28ba70e049d29bb281e6c614404e0c5049aDone https://github.com/home-assistant/core/pull/58699/commits/cfb0f28ba70e049d29bb281e6c614404e0c5049a